### ISSUE TITLE
Permission management

### ISSUE THREAD

**Issue by dnws** [microsoft/banner-settings-ado-extension#6]
Hello!

The extensions does exactly what it states, thanks for sharing it! However, I have a question about permissions.
In code I see no explicit permission validation, if user actually being a member of Project Collection Administrators (PCA) group or not, but verified it is working, as expected, only for PCAs. Do i get it right that code just relies on fact that API call will be failing if personal token gets rejected by some ADO internal permission policies? Or am I missing something?

Thanks

**Comment by j-windsor** [comment]
That is correct. As of now, the permission model relies on a failing request. Good suggestion! We can eventually get around to this, but I encourage you to implement if you have a few hours.
